x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities.tsx

```diff
@@ -4,17 +4,27 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { EuiCallOut, EuiSpacer } from '@elastic/eui';
 import React from 'react';
 
+import { LATEST_VULNERABILITIES_INDEX_DEFAULT_NS } from '../../../common/constants';
+import { useLatestFindingsDataView } from '../../common/api/use_latest_findings_data_view';
+import { useCspSetupStatusApi } from '../../common/api/use_setup_status_api';
+import { CloudPosturePage } from '../../components/cloud_posture_page';
+import { NoVulnerabilitiesStates } from '../../components/no_vulnerabilities_states';
+import { VULNERABILITIES_CONTAINER_TEST_SUBJ } from '../../components/test_subjects';
+
 export const Vulnerabilities = () => {
+  const dataViewQuery = useLatestFindingsDataView(LATEST_VULNERABILITIES_INDEX_DEFAULT_NS);
```

Comment:
it's ready to switch for `useFilteredDataView` now

Reply:
@opauloh I don't see `useFilteredDataView` in main, so I'm not sure what you are referring to